streaming/index.js

```diff
@@ -913,40 +1016,18 @@ const startServer = async () => {
     res.end(JSON.stringify({ error: 'Not found' }));
   };
 
-  app.use(setRequestId);
-  app.use(setRemoteAddress);
-  app.use(allowCrossDomain);
+  const api = express.Router();
 
-  app.get('/api/v1/streaming/health', (req, res) => {
-    res.writeHead(200, { 'Content-Type': 'text/plain' });
-    res.end('OK');
-  });
+  app.use(api);
```

Comment:
Ideally all of the following code would only happen for known eventsource streaming connections, but the spec doesn't allow us the ability to ensure that:
> User agents _**may [set](https://fetch.spec.whatwg.org/#concept-header-list-set)**_ (`[Accept](https://httpwg.org/specs/rfc7231.html#header.accept)`, `[text/event-stream](https://html.spec.whatwg.org/multipage/iana.html#text/event-stream)`) in request's [header list](https://fetch.spec.whatwg.org/#concept-request-header-list).

emphasis my own: https://html.spec.whatwg.org/multipage/server-sent-events.html

(so basically we can't add middleware that checks for the `Accept: text/eventsource` because the spec didn't mandate that clients send that header)